### DISCUSSION TITLE
No information available - no api response

### DISCUSSION THREAD

**Discussion by dbaanils** [deepgram/community#816] [General help] [answered]
### Which Deepgram product feature are you using?

Deepgram API - TTS

### Details

Hi Team,

Need urgent help , we are not getting any response from deepgram api and in console I can see it says status code as 403 and no information available flag is there!

the request id is 13e06a80-54da-477b-a76b-48c5af7e1240 ,  this is not only with this but many other continuously 

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

https://api.deepgram.com/v1/listen

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

13e06a80-54da-477b-a76b-48c5af7e1240 

### If possible, please attach your code or paste it into the text box.

randomNumber;
      videoOtpResponse.otp = randomNumber.toString();
      var uniqueID = document.getElementById('uniqueID');
      if (uniqueID != null) uniqueID.innerHTML = this.guId.toString();
      videoOtpResponse.key = this.guId.toString();
      $('#hedUniqueId').text(this.guId.toString());
      
      if (mediaDevice == 'iphone') {
      

        if (socket) {
          socket.send(JSON.stringify({ terminate_session: true }));
          socket.close();
          socket = null;
        }
        
        socket = new WebSocket('wss://api.deepgram.com/v1/listen', [
          'token',
          '',
        ]);
        socket.onopen = () => {
        

          var messageVar = document.getElementById('message');
          messageVar!.innerHTML = 'Please tell OTP which is show on screen.';
          messageVar!.style.display = 'block';
          mediaRecorder.addEventListener('dataavailable',

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by dbaanils** [comment]
I have already included all the details,

Also, another issue is  for four numbers transcript it is taling more then 10 seconds,

please provide a solution on the same!
5435a684-8d7a-4bf1-bda7-be1837613ca8

**Comment by SandraRodgers** [comment]
A 403 status code implies that there is a permissions issue. Could you try making a request in the playground with the same API key to see if it works? https://playground.deepgram.com/?endpoint=listen&smart_format=true&language=en&model=base

**Comment by jkroll-deepgram** [comment, marked as answer]
@dbaanils, looking into request ID `13e06a80-54da-477b-a76b-48c5af7e1240`, it appears that Deepgram returned a 408 Request Timeout, due to slow audio upload, which is typically a client-side networking limitation. I'm only seeing four 408s for your Deepgram project, which occurred June 15th-18th, out of ~200 total requests during that period, so this seems to be a transient issue. It's best practice to retry requests that do fail.

For that one request, you were sending a mp4 video file. One general option to improve upload speed is by only sending audio to Deepgram, rather than video.
